docs/introduction.mdx

```diff
@@ -109,6 +109,19 @@ Available Targets:
   help               - Display this help message
 ```
 
+## Crawling Web Pages
+
+By default, when passing in a URL to create a new DataSource for a Copilot, 15 pages will be indexed. The configuration for this is located in the `opencopilot/llm-server/utils/llm_consts.py` file:
+
+```python
+max_pages_to_crawl = int(os.getenv("MAX_PAGES_TO_CRAWL", "15"))
+```
+
+To increase the max pages to crawl, you can change the configuration file. However, this will create uncommitted changes in your working copy which you will need to stash in order to `git pull` for future updates. Another way to increase the max pages to crawl is to set an environment variable:
+
+```bash
+export MAX_PAGES_TO_CRAWL="50"
```

Comment:
I believe adding the `"` here is correct.
For example `export MAX_PAGES_TO_CRAWL="50"` vs `export MAX_PAGES_TO_CRAWL=50`

Testing this out in the same terminal gives the following response:

```bash
python3.11
Python 3.11.6 (main, Oct  2 2023, 13:45:54) [Clang 15.0.0 (clang-1500.0.40.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.getenv("MAX_PAGES_TO_CRAWL")
'150'
```

**User Interface**
This relates to viewing the paginated data sources e.g. `http://localhost:8888/copilot/0x_MY_DATA_SOURCES_1234/knowledge`

Just an FYI; the UI might need to be updated also because while the logs are showing many more than 15 pages being indexed the UI always shows 15:

![Screenshot 2024-02-19 at 11 39 51](https://github.com/openchatai/OpenCopilot/assets/9831342/6157633d-309a-4d2e-9e3e-f87577cb96e7)

Reply:
yup you're right, when we add to bash directly, we should use double quotes